content/client-lib-development-guide/features.textile

```diff
@@ -232,6 +232,22 @@ h3(#rest-auth). Auth
 ** @(RSA16b)@ If the library is provided with a token without the corresponding @TokenDetails@, then this holds a @TokenDetails@ instance in which only the @token@ attribute is populated with that token string
 ** @(RSA16c)@ Is set with the current token (if applicable) on instantiation and each time it is replaced, whether the result of an explicit @Auth#authorize@ operation, or a library-initiated renewal resulting from expiry or a token error response
 ** @(RSA16d)@ Is empty if there is no current token, including after a previous token has been determined to be invalid or expired, or if the library is using basic auth
+* @(RSA17)@ @Auth#revokeTokens@ function: This function is intended to be used to revoke tokens using provided @TokenRevocationRequest@ details. It internally creates request for  "REST revokeTokens endpoint":/rest-api/#revokeTokens and handles the response.
+** @(RSA17a)@ Method signature is @revokeTokens(TokenRevocationRequest)@.
+** @(RSA17b)@ If the @TokenRevocationRequest#key@ attribute is not provided, then the function must throw an error with a @"Key is not provided"@ message.
+** @(RSA17c)@ If the @TokenRevocationRequest#targets@ attribute is null or invalid (as per @TF3@ ), then the function must throw an error with a @"targets are invalid"@ message.
+** @(RSA17d)@ If the @TokenRevocationRequest#key@ is provided and @TokenRevocationRequest#targets@ valid, then the function must create the request and execute it.
+** @(RSA17e)@ Function returns a @TokenRevocationResponse@ object that contains the result of the request if request is successful.
+** @(RSA17f)@ Function throws an appropriate error/exception  if the request fails either for the reasons of invalid request, or as  a result of request failing.
```

Comment:
Tiny point but you have have an additional space before `if the request`

Reply:
1ceadf2c30128fefecfd387c89da7dd94e3724b6